src/blocks/TitleParagraphBlockBrandPivot.tsx

```diff
@@ -0,0 +1,69 @@
+import styled from '@emotion/styled'
+import React from 'react'
+import {
+  ContentWrapper,
+  MOBILE_BP_DOWN,
+  SectionWrapper,
+} from '../components/blockHelpers'
+import {
+  BrandPivotBaseBlockProps,
+  MarkdownHtmlComponent,
+} from './BaseBlockProps'
+
+const Headline = styled('h4')({
+  textTransform: 'uppercase',
+  fontSize: '1rem',
+  margin: 0,
+  [MOBILE_BP_DOWN]: {
+    textAlign: 'center',
+  },
+})
+
+const Paragraph = styled('div')({
+  display: 'block',
+  maxWidth: '100%',
+  width: '100%',
+  fontSize: '2rem',
+  lineHeight: '125%',
```

Comment:
Let's use unit less line-height values :) 
```suggestion
  lineHeight: 1.25,
```